### PR TITLE
Merge: Map feature after solving merge issues

### PR DESCRIPTION
**Pull request by aminJayP**

### CODE REVIEW

**Comment by AkhilSamarth** [comment]
PR currently has merge conflicts and does not pass testing

**Review by svp62 — APPROVED** [review]

File: package.json
Comment (Hammad-Akbar) [inline package.json:41]: Is this a dependency we need to install? If so add it to the readme.

**Review by Hammad-Akbar — APPROVED** [review]
Approving but after merging add the dependency we need to install to the readme

**Pull request merged**
